swift-sdk/uicomponents/IterableEmbeddedView.swift

```diff
@@ -299,6 +299,9 @@ public class IterableEmbeddedView:UIView {
                 imgView.isHidden = EMimage == nil
                 imgView.isHidden = self.EMimage == nil
                 imgView.image = EMimage
+                if !imgView.isHidden {
+                    imgView.widthAnchor.constraint(equalToConstant: 100).isActive = true
```

Comment:
This overrides the behavior by placing a constant 100 width for banner images if they are available